### PR TITLE
Add IStorage::FormatTmpPath static method

### PR DESCRIPTION
**Pull request by def-**
Not sure I prefer it, but now it's done. Any opinions?

## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: src/test/test.h [hunk 1]
```diff
@@ -9,6 +11,6 @@ class CTestInfo
 	CTestInfo();
 	IStorage *CreateTestStorage();
 	void DeleteTestStorageFilesOnSuccess();
-	char m_aFilename[64];
+	char m_aFilename[IO_MAX_PATH_LENGTH];
```
Comment (heinrich5991) [inline src/test/test.h:14]: Leaving it at 64 forgoes the include of base/system.h, I'd prefer that. It's no user-given paths anyway.

**Review by heinrich5991 — CHANGES_REQUESTED** [review]
Looks fine to me.

The update of `ddnet-libs` seems to be accidental.

**Comment by def-** [comment]
> The update of ddnet-libs seems to be accidental.

Yes, happens all the time to me since I have to use other ddnet-libs locally.

File: src/test/test.h [hunk 2]
```diff
@@ -1,6 +1,8 @@
 #ifndef TEST_TEST_H
 #define TEST_TEST_H
 
+#include <base/system.h>
```
Comment (heinrich5991) [inline src/test/test.h:4]: Include still there.

**Review by heinrich5991 — APPROVED** [review]
bors r+

Thanks.

**Comment by heinrich5991** [comment]
bors retry

**Pull request merged**